docs/scos/dev/guidelines/performance-guidelines/spryker-elastic-computing/spryker-elastic-computing.md

```diff
@@ -0,0 +1,308 @@
+---
+title: Spryker Elastic Computing
+description: This guideline explains how to use Elastic Computing functionality.
+last_updated: May 16, 2022
+template: concept-topic-template
+---
+
+Spryker provides several elastic computing features OOTB that allows to utilise computational resources in a more efficient way. 
+
+# Monitoring
+For enabling New Relic monitoring for P&S and queue workers infrastructure, please follow the next steps:
+
+1. Update `spryker/monitoring` module at least to v2.5.0.
+2. Add `queue:task:start` command to the argument grouped transactions in `Pyz\Zed\Monitoring\MonitoringConfig` configuration class
+```php
+namespace Pyz\Zed\Monitoring;
+
+use Spryker\Zed\Monitoring\MonitoringConfig as BaseMonitoringConfig;
+
+class MonitoringConfig extends BaseMonitoringConfig
+{
+    /**
+     * @api
+     *
+     * @return array<string>
+     */
+    public function getGroupedByArgumentTransactions(): array
+    {
+        return ['queue:task:start'];
+    }
+}
+```
+3. Add to the `Pyz\Zed\Monitoring\Business\MonitoringBusinessFactory` following lines of code:
```

Comment:
Explain what this does

Reply:
Enabling `FirstArgumentMonitoringConsoleTransactionNamingStrategy` to be used for transaction naming.